README.md

```diff
@@ -6,14 +6,13 @@
 
 <p align="center">
 <img src="https://app.bitrise.io/app/df00af454f27891d.svg?token=spjxNvzjnRqug6GfGM3_Lg"/>
-<img src="https://img.shields.io/cocoapods/v/WeScan.svg?style=flat"/>
 <img src="https://img.shields.io/cocoapods/l/WeScan.svg?style=flat"/>
```

Comment:
Not sure if we should remove these other "cocoapods" shields, too?

Reply:
I think the platform one and the license are nice to have because they don't mention anything about cocoapods. Let me know if you agree.